gravitee-node-api/src/main/java/io/gravitee/node/api/secrets/model/SecretURL.java

```diff
@@ -36,15 +44,16 @@ public record SecretURL(String provider, String path, String key, Multimap<Strin
      * Pair are always list as can be specified more than once. If no value is parsed, then <code>true</code> is set</li>
      *
      * @param url the string to parse
+     * @param includesSchema to indicate if URL should contain secret://
      * @return SecretURL object
      * @throws IllegalArgumentException when failing to parse
      */
-    public static SecretURL from(String url) {
+    public static SecretURL from(String url, boolean includesSchema) {
```

Comment:
```suggestion
    public static SecretURL from(String url, boolean includesScheme) {
```
What do you mean by Schema here? Shouldn't it be scheme?